tensorflow/core/kernels/data/experimental/data_service_dataset_op.cc

```diff
@@ -63,6 +49,22 @@ const int64 kRetryTimeoutMicros = 1000LL * 1000 * 60 * 60;  // 60 minutes.
 // Default interval between task list refreshes.
 const int64 kDefaultTaskRefreshIntervalMs = 1000;  // 1 second.
 
+//Tom: Avoid declare static const variables in class, which will result in import error (Symbol not found in flat namespace) in macOS. 
+// Define them in CC file instead.
+  static constexpr const char* const kDatasetType = "DataService";
```

Comment:
No need to declare 'static' in an anonymous namespace.

Reply:
I removed 'static' and unnecessary comments.